tests/scalars/spec.ts

```diff
@@ -27,17 +27,15 @@ describe('Custom scalar generation using casual', () => {
         expect(result).toBeDefined();
 
         // String
-        expect(result).toContain(
-            "str: overrides && overrides.hasOwnProperty('str') ? overrides.str! : 'ea corrupti qui incidunt eius consequatur blanditiis',",
-        );
+        expect(result).toMatch(/str: overrides && overrides\.hasOwnProperty\('str'\) \? overrides\.str! : '[^']+',/);
 
         // Float
-        expect(result).toContain(
-            "flt: overrides && overrides.hasOwnProperty('flt') ? overrides.flt! : -24.509902694262564,",
+        expect(result).toMatch(
```

Comment:
Makes sens for this kind of check, as long as we have a snapshot to validate the generated data are the same, I'm fine with regexp here 👍

Reply:
Yeah good point that the snapshots validate the other part :)